src/ring_theory/jacobson_ideal.lean

```diff
@@ -5,6 +5,36 @@ Authors: Kenny Lau, Devon Tuma
 -/
 import ring_theory.ideal_operations
 
+/-!
+# Jacobson Radical
+
+The Jacobson Radical of a ring `R` is defined to be the intersection of all maximal ideals of `R`.
```

Comment:
```suggestion
The Jacobson radical of a ring `R` is defined to be the intersection of all maximal ideals of `R`.
```
Maybe make the capitalization "Jacobson radical" throughout, for consistency (except, of course, when quoting Lean declarations).